client/src/slate-helpers/slate-utils/getInlinesFromNodeAndDescendants.ts

```diff
@@ -0,0 +1,13 @@
+export function getExportsFromNodeAndDescendants(node: any, pointerExportsSoFar: any[] = []) {
```

Comment:
This reads like you're mutating an input.

Reply:
Yeah good catch. This went through some iterations and I somehow missed this. I'll clean it up.

Reply:
a specific type declaration for pointerExportsSoFar, and maybe even for node, would be useful.